CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 2.6.0
+- The time steps do now work with the time helpers in ActiveSupport 4.1+ if the timecop gem is not included in the bundle.
+- The steps in the file `spreewald/timecop.rb` file were moved to `spreewald/time.rb` and importing `spreewald/timecop` directly is deprecated now.
+
```

Comment:
Please add another line explaining that the time steps will now actually _freeze_ the time, not only change it.